### PR TITLE
arange #6234

### PR DESCRIPTION
**Pull request by hslau44**

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/name_space_functions.py [hunk 1]
```diff
@@ -314,4 +319,4 @@ def bincount(x, weights=None, minlength=0, *, length=None):
 def cumprod(a, axis=0, dtype=None, out=None):
     if dtype is None:
         dtype = ivy.uint8
-    return ivy.cumprod(a, axis, dtype=dtype, out=out)
+    return ivy.cumprod(a, axis, dtype=dtype, out=out)
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/jax/numpy/name_space_functions.py:322]: A new line here to fix lint error
```suggestion
    return ivy.cumprod(a, axis, dtype=dtype, out=out)

```

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 2]
```diff
@@ -1507,6 +1507,39 @@ def test_jax_numpy_power(
     )
 
 
+# arange
+@handle_cmd_line_args
+@given(
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), num_arrays=3, shape=(1,), shared_dtype=True, min_value=0, max_value=100,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1514]: We don't need to specify the range you can remove the `max_value` and `min_value`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hi @hslau44 

Thanks for the contribution 😊

Just a few small changes are to be made and we merge your PR.

Thanks🙂

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Still, some lint errors are occurring try to update with the master branch and fix all the lint errors in the `test_jax_numpy_namespace_functions.py` file that are being caused by your changes after that I will merge your PR.

Thanks:)

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py [hunk 3]
```diff
@@ -1507,6 +1507,39 @@ def test_jax_numpy_power(
     )
 
 
+# arange
+@handle_cmd_line_args
+@given(
+    dtype_and_values=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"), num_arrays=3, shape=(1,), shared_dtype=True,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_namespace_functions.py:1514]:
```suggestion
        available_dtypes=helpers.get_dtypes("numeric"),
        num_arrays=3,
        shape=(1,),
        shared_dtype=True,
```

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]

**Review by hirwa-nshuti — APPROVED** [review]
Lgtm!

**Comment by hslau44** [comment]
thanks!

**Pull request merged**